src/index.html

```diff
@@ -12,6 +12,8 @@
 
 	<!-- inject:css -->
 	<!-- endinject -->
+
+	<script src="/js/app.config.js"></script>
```

Comment:
Probably best calling it `app.loader.(ts|js)` and prevent it from entering the bundled file, so it can always be included in the `index.html`.